docs/configuration.asciidoc

```diff
@@ -455,7 +455,7 @@ See https://www.owasp.org/index.php/Information_exposure_through_query_strings_i
 [[config-global-labels]]
 ==== `GlobalLabels` (added[1.2])
 
-Labels added to all events, with the format `key=value[,key=value[,...]]`.
+Labels added to all events, with the format `"GlobalLabels": "key=value,key=value,..."`.
```

Comment:
I don't think the format should include `"GlobalLabels"`, as it's not correct for a value coming from environment variables, or certain `IConfiguration` structures [such as from ASP.NET Full Framework's appSettings](https://www.elastic.co/guide/en/apm/agent/dotnet/master/configuration-on-asp-net.html#asp-net-sample-config).